partition/bootloader.go

```diff
@@ -146,3 +155,13 @@ func MarkBootSuccessful(bootloader Bootloader) error {
 
 	return bootloader.SetBootVars(m)
 }
+
+func reboot() error {
+	cmd := exec.Command("shutdown", "+10", "-r", shutdownMsg)
```

Comment:
given we are doing this, can we instead of hard coding +10 here, pass in a afterMins params to Reboot(afterMins int)